settings.gradle.kts

```diff
@@ -1 +1,2 @@
 include("api")
+include("web")
```

Comment:
`include` function takes vararg as an argument, so It's possible to enumerate all modules In one `include`. I don't know which approach is better, but I usually use single `include`

Reply:
Thanks!